server/src/server.py

```diff
@@ -26,5 +27,10 @@
 api.add_resource(Bets, "/bets")
 api.add_resource(Bet, "/bets/<string:bet_id>")
 
+# Points APIs
+api.add_resource(Point, "/points")
+api.add_resource(Points, "/points/<string:user_id>")
```

Comment:
To be consistent, Let's use the resource "Points" to handle the get all points (leaderboard) and "Point" for points apis associated with a specific user